### PR TITLE
PLUGIN wallabag

### PR DESCRIPTION
**Pull request by ArthurHoaro**
Add a Wallabag icon in linklist for each link.

### CODE REVIEW

File: plugins/wallabag/wallabag.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+// don't raise unnecessary warnings
+if (is_file(PluginManager::$PLUGINS_PATH . '/wallabag/config.php')) {
+    include PluginManager::$PLUGINS_PATH . '/wallabag/config.php';
+}
+
+if (!isset($GLOBALS['plugins']['WALLABAG_URL'])) {
+    header('Content-Type: text/plain; charset=utf-8');
```
Comment (virtualtam) [inline plugins/wallabag/wallabag.php]: seems a bit harsh to redirect the user to an error page

maybe we could introduce an helper function in the plugin manager to display a summary of current config errors in a dedicated `<div>` when the user is logged in?

File: plugins/wallabag/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+## Save to Wallabag plugin for Shaarli
```
Comment (virtualtam) [inline plugins/wallabag/README.md]: \+ MIT license for the Wallabag icon - https://github.com/wallabag/wallabag/blob/master/COPYING.md

File: plugins/wallabag/wallabag.php (hunk 1 above)
Comment (ArthurHoaro) [inline plugins/wallabag/wallabag.php]: I'm opening an issue on this that should be fixed before the release.

File: plugins/wallabag/README.md (hunk 2 above)
Comment (ArthurHoaro) [inline plugins/wallabag/README.md]: Thank you, fixed.

**Pull request merged**